src/load_order.ts

```diff
@@ -478,28 +480,9 @@ export const makeV2077LoadOrderFrom = (
 };
 
 
-export const loadOrderToREDdeployRunParameters = (
+export const redmodDeployRunParameters = (
   gameDirPath: string,
-  v2077LoadOrderToDeploy: LoadOrder,
 ): VortexRunParameters => {
-  const v2077LoadOrderEntries =
-    v2077LoadOrderToDeploy.entriesInOrderWithEarlierWinning;
-
-  const loadOrderForREDmodDeployWithShellQuotes = pipe(
-    v2077LoadOrderEntries,
-    filterMap((mod) =>
-      (mod.enabled
-        ? some(`"${path.basename(mod.redmodPath)}"`)
-        : none)),
-  );
-
-  const loadOrderedModListToDeploy =
-    isEmpty(loadOrderForREDmodDeployWithShellQuotes)
```

Comment:
I think it's been working with the default deploy because of the `-force`?

Reply:
ohhh... that makes sense. Let me test that then real quick

Reply:
Hmm, not seeing that happen. maybe the command changed? When forcing deploy with empty file, `-mod=` with no mods, or without the option completely it just runs the regular deploy using the existing load-order defined in mods.json

Reply:
Yep, that sounds like it.

I think we can punt on this, too. The 'right' choice for our purposes might be to delete mods.json if the LO drops to 0, but this'll match previous behavior.